tutorials/casper/README.md

```diff
@@ -0,0 +1,57 @@
+
+# Casper 
+
+```bash
+module purge
+module load gnu/9.1.0 ncarcompilers openmpi netcdf ncarenv cmake
+```
+
+I also needed a newer version of gmake, it's in /glade/work/jedwards/make-4.3/bin/make
```

Comment:
``conda install make`` will also provide a new version of ``make`` for the user that will satisfy this requirement